README.md

```diff
@@ -85,6 +85,12 @@ Integration tests can be run on any deployed DC/OS cluster. For installation ins
 
 Integration tests are installed via the [dcos-integration-test](./packages/dcos-integration-test/) Pkgpanda package.
 
+Integration test files are stored on the DC/OS master node at `/opt/mesosphere/active/dcos-integration-test`.
+Therefore, in order to test changes to test files, move files from `packages/dcos-integration-test/extra/` in your checkout to `/opt/mesosphere/active/dcos-integration-test` on the master node.
+
+The canonical source of the test suite's results is the continuous integration system.
+There may be differences between the results of running the integration tests as described in this document and the results given by the continuous integration system.
```

Comment:
In my mind, this may be also:
* results are presented differently
* results are incomplete
* results differ as per some stuff fails in CI while it does not in case of local run
* results differ as per some tests are not run in the CI

Do you see what I mean?